notify/mail.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env sh
 
-# support local mail app
+#Support local mail app
+
+#MAIL_FROM="yyyy@gmail.com"
+#MAIL_TO="yyyy@gmail.com"
 
```

Comment:
we need a public `MAIL_BIN` variable.   The user can choose which app to use, if both `mail` and `sendmail` are installed.

The MAIL_BIN variable is optional.

Reply:
Done.